Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -130,7 +130,7 @@ def audit
       formula_count += 1
       problem_count += fa.problems.size
       problem_lines = format_problem_lines(fa.problems)
-      corrected_problem_count = options[:style_offenses]&.count(&:corrected?)
+      corrected_problem_count = options[:style_offenses]&.count(&:corrected?).to_i
```

Comment:
How about `options.fetch(:style_offenses, []).count(&:corrected?)`?

Reply:
I like that, yeh 👍

Reply:
Updated. Reads much better, thanks!